cloudgenix_config/default_interfaces.py

```diff
@@ -1229,6 +1229,563 @@
             used_for: none
 """
 
+ion_1000 = """
+          '1':
```

Comment:
We missed adding the new interface attributes to the default configs of ions 2k, 3k and 7k.